src/containers/Profile/index.js

```diff
@@ -115,21 +115,6 @@ class ProfileComponent extends React.Component {
   componentDidMount() {
     const body = document.getElementsByClassName('page-body')[0];
     body.addEventListener('scroll', this.onScroll);
-
-    const {
```

Comment:
is there a reason why this is getting removed? maybe you haven't rebased against master?